api_app/migrations/0003_auto_20201020_1406.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.14 on 2020-10-20 14:06
+
+from django.db import migrations
+from datetime import timedelta
+
+
+def create_default_clients(apps, schema_editor):
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Client = apps.get_model("durin", "Client")
+    # for pyintelowl, custom token_ttl
+    Client.objects.update_or_create(name="pyintelowl", token_ttl=timedelta(days=365))
```

Comment:
is it possible to customize this time to live? from the environment variables? I would set the default to "no expiration" if possible. In this way the pyintelowl token for a user would mimic a classic Token

Reply:
We can make it customizable, yes through an environment variable. I'll do that. 
However, I am struggling to have an option for "no expiration" because that introduces too many if conditions (hence bugs) which are quite hard to test. We could change the value to  2, 5 or even 10 years, which essentially is "no expiration" I guess.

Reply:
Yes you are right, a 100 years value I think is enough :P Where will IntelOwl be in 2120? :thinking:

Reply:
In 100 years ? I just wish I am not still working on authentication. xD

Reply:
no well, jokes aside, I think that we could set >=10 years like you mentioned for this so it is like "no expiration"